docs/readmes/nms/alert.md

```diff
@@ -0,0 +1,322 @@
+### **Alarm component’s Alert Tab**
+
+Alerts can also be viewed from the alert tab in the Alarm table.
+![viewing_alerts_2](assets/nms/userguide/alerts/viewing_alerts2.png)
+![viewing_alerts_3](assets/nms/userguide/alerts/viewing_alerts3.png)
+
+## Alert Receivers
+
+Alert Receiver is created to push the alert notification in real time so that the operator is notified instantaneously when the alert is fired. Following example details the steps involved in creating a slack alert receiver and configuring it in NMS.
+
+### Example: Adding Slack Channel as Alert Receiver
+
+**Generate Slack Webhook URL:**
+
+* Create an App: Goto https://api.slack.com/apps?new_app=1 and click on “Create New App”. Enter the App Name and the Slack Workspace.
+* Click on “Incoming Webhooks” and change “Active Incoming Webhooks” to On
+
+![alert_recv1](assets/nms/userguide/alerts/alert_recv1.png)
+* Scroll down and create a new Webhook by clicking on “Add New Webhook to Workspace”. Select the Slack Channel name.
+
+![alert_recv1](assets/nms/userguide/alerts/alert_recv2.png)
+* Copy the “Webhook URL” once it is generated.
+
+![alert_recv1](assets/nms/userguide/alerts/alert_recv3.png)
+**Create a new Alert Receiver in NMS:**
+![alert_recv1](assets/nms/userguide/alerts/alert_recv4.png)
+
+**Testing the newly added alert receiver**
+
+<TBD>
+
+## Configuring Alert rules
+
+Alert rule configuration lets us define rules for triggering alerts.
+
+### Predefined Alerts
+
+Magma NMS comes loaded with some default set of alert rules. These default rules aren’t configured automatically. If operator chooses to use these default rules, they can do it by clicking “Sync Predefined Alerts” button in Alert rule configuration tab. As shown below
+![alerts1](assets/nms/userguide/alerts/alerts1.png)![alerts2](assets/nms/userguide/alerts/alerts2.png)Currently predefined alerts configure following alerts
+
+* CPU percent on the gateway is running > 75% in last 5 minutes
+* Unsuccessful S1 setup in last 5 minutes
+* S6A authorization failures in last 5 minutes
+* Upon exceptions when bootstrapping a gateway in last 5 minutes
+* When services were restarted unexpectedly in last 5 minutes
+* When a UE attach resulted in a failure in last 5 minutes
+* When there were any service restarts in last 5 minutes
+
+**Note:** Operator will have to go and additionally specify the receiver in each of these alerts when they want to be notified of the alerts as follows,
+![alerts3](assets/nms/userguide/alerts/alerts3.png)
+
+### Custom Alert Rules
+
+Operator can create custom alert rules by applying creating a expression based on metrics.
+
+**Metrics Overview**
+Magma gateways collect various metrics at a regular intervals and push them into Orchestrator. The Orchestrator stores these metrics in a prometheus instance(implementation detail). The prometheus instance along with alert manager provides us support in querying various metrics on the system and setting alerts based on that.
```

Comment:
I'm not sure about saying prometheus is an implementation detail here since the entire alerting framework relies on using prometheus. At this point it's not really a detail and more of an integral part of orc8r. Also Prometheus should probably be capitalized.